ComputeClient/Compute.Client/ApiUris.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using System.Data;
-using System.Text; 
+using System.Text;
 
```

Comment:
Real changes start on line 3039. Other changes are as a result of formatting in Visual Studio 2017.